libs/components/forms/src/lib/modules/file-attachment/file-attachments.module.ts

```diff
@@ -37,6 +41,7 @@ import { SkyFileSizePipe } from './file-size.pipe';
     SkyFileDropComponent,
     SkyFileItemComponent,
     SkyFileSizePipe,
+    SkyFormErrorModule,
```

Comment:
`SkyFormErrorModule` doesn't need to be exported from this module. You only need to export a module from another module when the first module represents a subset of features of the second module. `SkyFormErrorModule` is already a standalone module available for the consumer to import.